FindEth.cmake

```diff
@@ -19,20 +19,7 @@ endif()
 set(ALL_LIBS ${CORE_LIBS};evmjit;solidity;secp256k1;testutils)
 
 set(ETH_INCLUDE_DIRS ${ETH_INCLUDE_DIR})
-set(ETH_CORE_LIBRARIES ${ETH_LIBRARY})
-
-foreach (l ${ALL_LIBS})
-	string(TOUPPER ${l} L)
-	find_library(ETH_${L}_LIBRARY
```

Comment:
cmake uses _target_ dependencies, not file dependencies. As long as e.g. "testutils" is a cmake target it will know where the library is.